catalog/core/catalog-core-solr/src/main/java/ddf/catalog/source/solr/SolrCatalogProvider.java

```diff
@@ -304,6 +358,11 @@ public UpdateResponse update(UpdateRequest updateRequest) throws IngestException
                         "Found more metacards than updated metacards provided. Please ensure your attribute values match unique records.");
             }
 
+        } else if (pendingNrtIndex.getAllPresent(identifiers).size() > 0) {
+            Map<String, Metacard> matches = pendingNrtIndex.getAllPresent(identifiers);
```

Comment:
Not want to deal with having to store this in a variable instead of fetching it twice?